pom.xml

```diff
@@ -0,0 +1,162 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.dbpedia.spotlight</groupId>
+  <artifactId>wikipedia-stats-extractor</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+  </licenses>
+  <properties>
+    <scala.version>2.10.4</scala.version>
+      <project.source.charset>UTF-8</project.source.charset>
+      <project.source.encoding>UTF-8</project.source.encoding>
+      <java.version.source>1.7</java.version.source>
+      <java.version.target>1.7</java.version.target>
+  </properties>
+
+  <repositories>
+    <repository>
+      <id>scala-tools.org</id>
+      <name>Scala-Tools Maven2 Repository</name>
+      <url>http://scala-tools.org/repo-releases</url>
+    </repository>
+      <repository>
+          <id>maven.aksw.internal</id>
+          <name>University Leipzig, AKSW Maven2 Repository</name>
+          <url>http://maven.aksw.org/archiva/repository/internal</url>
+      </repository>
+  </repositories>
+
+  <pluginRepositories>
+    <pluginRepository>
+      <id>scala-tools.org</id>
+      <name>Scala-Tools Maven2 Repository</name>
+      <url>http://scala-tools.org/repo-releases</url>
+    </pluginRepository>
+  </pluginRepositories>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.scala-lang</groupId>
+      <artifactId>scala-library</artifactId>
+      <version>${scala.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.1</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.specs</groupId>
+      <artifactId>specs</artifactId>
+      <version>1.2.5</version>
+      <scope>test</scope>
+    </dependency>
+      <dependency>
+          <groupId>org.apache.spark</groupId>
```

Comment:
Yes using IDE. I have spark installed on my laptop. It is working fairly well with small datasets.